src/lib/sm.h

```diff
@@ -29,6 +29,8 @@ struct sm
 {
 	int rc;
 	int state;
+	const char *name;
+	unsigned long int id;
```

Comment:
i don't remember if it was introduced by me, still please use uint64_t here.

Reply:
Changed.